pkg/util/mount/exec_mount_test.go

```diff
@@ -94,85 +92,26 @@ func TestUnmount(t *testing.T) {
 		return nil, nil
 	})
 
-	wrappedMounter := &fakeMounter{t}
+	wrappedMounter := &fakeMounter{&FakeMounter{}, t}
 	mounter := NewExecMounter(exec, wrappedMounter)
 
 	mounter.Unmount(destinationPath)
 }
 
 /* Fake wrapped mounter */
 type fakeMounter struct {
-	t *testing.T
+	*FakeMounter
+	*testing.T
```

Comment:
keep the name lowercase?

Reply:
Done. `T` changed to `t`. (`fakeMounter` needs to embed `FakeMounter` to inherit methods so it cannot be lowered)